assets/src/admin-meeting.scss

```diff
@@ -98,6 +98,17 @@
 		}
 	}
 
```

Comment:
used `<small>` tags to follow input fields for immediate error messaging.

Reply:
👍 thanks for using the same color of red we use in other places - maybe eventually we can use css vars for it